src/components/standalone/firewall/CreateOrEditPortForwardDrawer.vue

```diff
@@ -165,13 +165,25 @@ watchEffect(() => {
 })
 
 function close() {
+  if (
+    error.value.notificationTitle === t('error.cannot_edit_port_forward') ||
+    error.value.notificationTitle === t('error.cannot_create_port_forward')
+  ) {
+    error.value.notificationTitle = ''
+    error.value.notificationDescription = ''
+  }
+  validationErrorBag.value.clear()
+  restrictIPValidationErrors.value = []
+
   resetForm()
   emit('close')
 }
 
 onMounted(() => {
   fetchOptions()
-  firewallConfig.fetch()
+  if (firewallConfig.loading || firewallConfig.error) {
```

Comment:
Can you explain this condition?

Reply:
Considering how `useFirewallStore` is a global store and is used in different pages other than the port forward one, I thought it wouldn't make much sense to refetch its items when they already loaded, hence why i've included this condition. If you think this may cause issues I can remove it though!

Reply:
> Considering how `useFirewallStore` is a global store and is used in different pages other than the port forward one, I thought it wouldn't make much sense to refetch its items when they already loaded, hence why i've included this condition. If you think this may cause issues I can remove it though!

Now it's clear, thanks